### ISSUE TITLE
CS4ZP6 Tester Feedback: Program Map Course Buttons do nothing

### ISSUE THREAD

**Issue by elwazana** [trevorNgo/Measure2.0#206]
Environment
- Google Chrome
- Windows 10

Describe the feedback/issue.
On the `Program Map` page programs have listed courses that are clickable but do nothing. 
![Screenshot 2021-03-06 151210](https://user-images.githubusercontent.com/35268928/110219624-66cb7e00-7e8e-11eb-951d-4766772bbe35.png)
 
Priority
LOW (I don't believe this is implemented yet)

Please list the reproduction steps
- Login as `Department Coordinator`
- Navigate to `Program Map`
- Try clicking any of the courses listed under a program

**Comment by trevorNgo** [comment]
yup, just recently added: https://github.com/trevorNgo/Measure2.0/pull/135
